dist.ini

```diff
@@ -0,0 +1,44 @@
+name = AnyEvent-RabbitMQ
+author = Masahito Ikuta <cooldaemon@gmail.com>
+license = Perl_5
+copyright_holder = AnyEvent-RabbitMQ's developers
+
+[@Filter]
+-bundle = @Basic
+-remove = Readme
+
+[Metadata]
+x_contributors = Tom Doran <bobtfish@cpan.org>
+x_contributors = Nicolas R <atoomic@cpan.org>
+x_contributors = Dave Lambley <dlambley@cpan.org>
+x_contributors = Ruslan Zakirov <ruz@bestpractical.com>
+x_contributors = Masahito Ikuta <cooldaemon@gmail.com>
+x_contributors = Rod Taylor <rod.taylor@gmail.com>
+x_contributors = Carl Hörberg <carl@cloudamqp.com>
+
+[InstallGuide]
+;[CPANFile]
+[VersionFromModule]
+[MetaJSON]
+
+[Repository]
+repository = git://github.com/bobtfish/AnyEvent-RabbitMQ.git
```

Comment:
https://metacpan.org/pod/Dist::Zilla::Plugin::MetaResources is core Dist::Zilla and has more options, better than another author dependency.